ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -381,7 +384,13 @@ function getValues(pendingApproval, t, actions, history, data) {
           nickname: pendingApproval.requestData.chainName,
         });
 
-        history.push(DEFAULT_ROUTE);
+        const locationPath = document.location.hash.replace('#', '/');
```

Comment:
Good fix.  I noticed this too but wasn't sure how to fix it.  I was trying use `state.metamask.completedOnboarding` to determine whether to redirect, but didn't know how to access it here.  document.location seems like it'll work